pallets/portal/Cargo.toml

```diff
@@ -27,6 +27,8 @@ pallet-grandpa-finality-verifier = { path = "../../finality-verifiers/grandpa",
 t3rn-abi        = { default-features = false, path = "../../types/abi", features = [ "runtime" ] }
 t3rn-primitives = { default-features = false, path = "../../primitives" }
 t3rn-types      = { default-features = false, path = "../../types", features = [ "runtime" ] }
+xcm              = { workspace = true }
```

Comment:
missing in std + try runtime?

Reply:
fixed in [#e7cf52c](https://github.com/t3rn/t3rn/pull/1469/commits/e7cf52c373fa7f4b1441f57c4cd707a49f65d3e2)